source/reference/pmm-admin.rst

```diff
@@ -0,0 +1,314 @@
+.. _pmm.ref.pmm-admin:
+
+###################################
+pmm-admin - PMM Administration Tool
+###################################
+
+****
+NAME
+****
+
+``pmm-admin`` - Administer PMM
+
+********
+SYNOPSIS
+********
+
+``pmm-admin [OPTIONS] COMMAND [FLAGS]``
+
+``pmm-admin add mongodb|mysql|postgresql|proxysql``
+
+``pmm-admin remove <service-type> [<service-name>]``
+
+``pmm-admin register``
+
+``pmm-admin status --server-url=<SERVER-URL>``
+
+``pmm-admin list --server-url=<SERVER-URL>``
+
+``pmm-admin register --server-url=<SERVER-URL>``
+   Register the current node with the PMM Server located at ``SERVER-URL``.
```

Comment:
`pmm-amin config`  - one of the most important lost in this section?

Reply:
Yes, unintentionally.